homeassistant/helpers/entity_platform.py

```diff
@@ -435,9 +437,14 @@ async def _async_add_entity(
                 already_exists = True
 
         if already_exists:
-            msg = f"Entity id already exists - ignoring: {entity.entity_id}"
             if entity.unique_id is not None:
-                msg += f". Platform {self.platform_name} does not generate unique IDs"
+                msg = f"Platform {self.platform_name} does not generate unique IDs "
```

Comment:
I suggest splitting the message into two parts with period.
```suggestion
                msg = f"Platform {self.platform_name} does not generate unique IDs. "
```

Reply:
Good call 👍